Terran-One/cw-simulate-ui
# On clicking of step the tab should be switched to State

## Issue link

[WL-492](https://terran-one.atlassian.net/browse/WL-492)

## Description

In state stepper, set the history active tab to state when clicking on a new item in the timeline.

## Test steps

1. Create an environment, navigate to simulation screen
2. Perform some executions
3. Navigate to request or response tab
4. Navigate to another node in the state stepper timeline
5. Observe that State is now selected
